.eslintrc.js

```diff
@@ -17,6 +18,11 @@ module.exports = {
     ecmaVersion: 'latest',
     sourceType: 'module',
   },
-  rules: {},
+  rules: {
+    'no-unused-vars': ['error', {
+      'argsIgnorePattern': '^_',
+      'varsIgnorePattern': '^_'
+    }],
+  },
```

Comment:
Stops prettier from throwing an error for unused vars that start with an underscore. Was necessary in Toolbar.js, because there was an unused var className. I renamed it to _className